applications/mobu/templates/configmap-github-ci-app.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.config.githubCiApp -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "mobu.fullname" . }}-github-ci-app
+  labels:
+    {{- include "mobu.labels" . | nindent 4 }}
+data:
+  yaml: |
```

Comment:
Nit: I would use the filename here rather than `yaml` as a key. Either way works, but the file name is a bit more self-documenting.